tactics/cbn.ml

```diff
@@ -569,6 +572,9 @@ let whd_state_gen ?csts flags env sigma =
   let open Context.Named.Declaration in
   let open ReductionBehaviour in
   let rec whrec cst_l (x, stack) =
+    let () = match stack with
```

Comment:
This looks like remnant of debug code.

Reply:
Fixed (and changelog at the same time).

In passing, do you know a more compact way than 5 lines to change the label of a constant? Should I make a function in the API of Names for it?

Reply:
There is `Names.Constant.change_label`.

Reply:
Thanks